### ISSUE TITLE
Enable GCP alerts during business hours

### ISSUE THREAD

**Issue by alex-dabija** [giantswarm/roadmap#1296]
## Task

Enable GCP alerts during business hours in order to get a feeling on what issues CAPG clusters have.

**Comment by alex-dabija** [comment]
Prometheus needs to have CSI working with workload identity. @alex-dabija create separate issue.

**Comment by alex-dabija** [comment]
The separate issue to have CSI working with workload identity: https://github.com/giantswarm/roadmap/issues/1453 .

**Comment by alex-dabija** [comment]
Blocked by: https://github.com/giantswarm/roadmap/issues/1453 .

**Comment by brinker211** [comment]
@alex-dabija this is done as I see the team working on the alerts but can you confirm and we close? Thanks!

**Comment by alex-dabija** [comment]
GCP alerts are enabled and working.
